docs/documentation/components/finn-large.html

```diff
@@ -0,0 +1,238 @@
+                </ul>
+              </li>
+            </ol>
+          </section>
+
+          <section id="examples">
+            <h1>Voorbeelden</h1>
+
+            <h2>Kleurenset voorbeeld</h2>
+            <h3>Visueel voorbeeld:</h3>
+            <ul class="branding">
+              <li><span class="branding-color-1">branding-color-1</span></li>
+              <li><span class="branding-color-2">branding-color-2</span></li>
+              <li><span class="branding-color-3">branding-color-3</span></li>
+              <li><span class="branding-color-4">branding-color-4</span></li>
+              <li><span class="branding-color-5">branding-color-5</span></li>
+            </ul>
+
+            <h3>CSS-voorbeeld</h3>
+            <pre>
+              <code>
+:root {
+  /* Branding color 1 */
+  --branding-color-1-background-color: WhiteSmoke;
+  --branding-color-1-text-color: black;
+
+  /* Branding color 2 */
+  --branding-color-2-background-color: SeaShell;
+  --branding-color-2-text-color: black;
+
+  /* Branding color 3 */
+  --branding-color-3-background-color: LightSeaGreen;
+  --branding-color-3-text-color: white;
+
+  /* Branding color 4 */
+  --branding-color-4-background-color: MediumVioletRed;
+  --branding-color-4-text-color: white;
+
+  /* Branding color 5 */
+  --branding-color-5-background-color: SlateBlue;
+  --branding-color-5-text-color: white;
+}
+              </code>
+            </pre>
+
+            <h2>Variabele gebruiken binnen een andere variabele</h2>
+            <h3>HTML-voorbeeld:</h3>
+            <pre>
+              <code>
+:root {
+  --button-base-background-color: var(--branding-color-accent-background-color);
+}
+              </code>
+            </pre>
+            <p>Voor meer informatie hierover zie <a href="../use-css-variable.html">CSS-variabelen gebruiken</a>.</p>
+
+            <h2>Variabele gebruiken binnen de html op een specifiek element</h2>
+            <p class="warning"><span>Waarschuwing:</span> Waar mogelijk is het altijd aan te raden zo min mogelijk classes te gebruiken maar om gebruik te maken van het overerven van CSS. Maar omdat een uitzondering soms nodig is, is het mogelijk om de huistijlkleuren ook direct in de html te kunnen definieren door middel van een classes.</p> 
+            <p>De knoppen binnen het "button-base"-bestand maken gebruik van de accentkleur als deze binnen het "application-base"-bestand gedifieerd staat. Voor meer informatie zie <a href="../use-css-variable.html">CSS-variabelen gebruiken</a>.</p>
+            
+            <h3>Visueel voorbeeld:</h3>
+            <div class="branding-color-1">
```

Comment:
As far as all these kind of visual examples go (for each color set): `branding-color-1` is almost exactly the same as `--code-base-background-color`, which is used for the code blocks. So it isn't a great visual example, because I thought it was a code block, instead of a `div` with `branding-color-1`. I would suggest using color 2 or choose another color for color 1, to make these visual examples stand out more.